src/NzbDrone.Core/DecisionEngine/DownloadDecisionComparer.cs

```diff
@@ -36,7 +36,6 @@ public int Compare(DownloadDecision x, DownloadDecision y)
                 CompareEpisodeNumber,
                 ComparePeersIfTorrent,
```

Comment:
Yes the extension methods at the moment hide the distance function, but we can add extension methods that expose the distance function rather than just a distance value selector. When we were talking about this originally we discussed `log`, specifically around peer counts. There is a test here that is a simple example:

https://github.com/Matty666/Sonarr/blob/5ce59602a34aac64d4aaec2ede63b0c80a13815c/src/NzbDrone.Core.Test/DecisionEngineTests/ClusterAnalysis/HierarchicalClusteringFixture.cs#L103